src/test/java/lithium/university/TwitterResourceTest.java

```diff
@@ -0,0 +1,68 @@
+package lithium.university;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+
+import javax.ws.rs.core.Response;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TwitterResourceTest {
+    private TwitterRetrieve twitterRetrieveTest;
+    private TwitterPublish twitterPublishTest;
+
+
+    @Before
+    public void init() {
+        twitterRetrieveTest = Mockito.mock(TwitterRetrieve.class);
+        twitterPublishTest = Mockito.mock(TwitterPublish.class);
+    }
+
+    private Status mockStatus() {
+        Status s = Mockito.mock(Status.class);
+        Mockito.when(s.getText()).thenReturn("This is a mocked status!");
+        return s;
+    }
+
+    @Test
+    public void testResourceGetTimeline() throws TwitterException {
+        TwitterResource twitterResourceTest = new TwitterResource(twitterRetrieveTest, twitterPublishTest);
+        List<Status> fakeList = new ArrayList<>();
+        fakeList.add(mockStatus());
+
+        Mockito.when(twitterRetrieveTest.retrieveFromTwitter(Mockito.any(Twitter.class), Mockito.anyInt())).thenReturn(fakeList);
+
+        Response response = twitterResourceTest.getHomeTimeline();
+        Assert.assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+        Assert.assertEquals("This is a mocked status!", ((List<Status>) ((Tweet) response.getEntity()).getContent()).get(0).getText());
```

Comment:
instead of having the text hardcoded.  I'd generally go with the mockStatus.getText().  In your scenario since it's in a mocked object, pulling it from a constant would be better.  Less prone to error long term.